### PR TITLE
[APPS-3459] Update en.json file with marketplace listing content

### PR DESCRIPTION
**Pull request by pmgrove**
:v:

/cc @zendesk/vegemite 

### Description

Update en.json file with marketplace listing content

### References
* JIRA: https://zendesk.atlassian.net/browse/APPS-3459

### Risks
* [low] - app content may not show on marketplace

### CODE REVIEW

File: dist/translations/en.json [hunk 1]
```diff
@@ -1,54 +1,55 @@
 {
   "app": {
-    "package": "app_name",
+    "name": {
+      "title": "Name for App: v2 REPL",
+      "value": "v2 REPL"
+    },
     "description": {
```
Comment (ocke) [inline dist/translations/en.json]: `short_description`?

File: dist/translations/en.json [hunk 2]
```diff
@@ -1,54 +1,55 @@
 {
   "app": {
-    "package": "app_name",
+    "name": {
```
Comment (ocke) [inline dist/translations/en.json]: name is not a thing.

I think it should be a thing. But right now, it's not a thing.

File: dist/translations/en.json (hunk 1 above)
Comment (nick-f) [inline dist/translations/en.json]: Is this related to the changes that @zendesk/wombat are making to the marketplace description? I've heard them talking about something similar but I don't remember specifics of it.

Comment (mtsolakiszen) [inline dist/translations/en.json]: Either description or short_description work. Should be fine 👍

**Pull request merged**